src/SFA.DAS.Commitments.Application.UnitTests/Commands/UpdateApprenticeshipStatus/WhenResumingAStartedApprenticeship.cs

```diff
@@ -121,6 +120,17 @@ public async Task ThenShouldSendAnApprenticeshipEventWithDateOfChange()
                 null));
         }
 
+        [Test]
+        public async Task ThenShouldPublishAV2ApprenticeshipResumedEvent()
+        {
+            await Handler.Handle(ExampleValidRequest);
+
+            MockV2EventsPublisher.Verify(x => x.PublishApprenticeshipResumed(
```

Comment:
It's worth verifying the call with the actual commitment and apprenticeship objects instead of `It.IsAny`.

Reply:
yes